__integrations__/credentials/VerifiableCredential.test.js

```diff
@@ -19,7 +21,7 @@ describe('Integration Tests for Verifiable Credentials', () => {
       { givenNames: 'Joao', otherNames: 'Barbosa', familyNames: 'Santos' });
 
     const dob = await Claim.create('claim-cvc:Identity.dateOfBirth-v1', { day: 20, month: 3, year: 1978 });
-    const cred = await VC.create('credential-cvc:Identity-v1', uuidv4(), null, [name, dob], 1);
+    const cred = await VC.create('credential-cvc:Identity-v3', uuidv4(), null, credentialSubject, [name, dob]);
```

Comment:
Would it not be better to leave v1 as a dedicated test and add v3 in order to prove backwards compatibility?